balancer/rls/internal/testutils/channel.go

```diff
@@ -0,0 +1,69 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package testutils provides utility types, for use in tests.
+package testutils
+
+import (
+	"errors"
+	"time"
+)
+
+// ErrRecvTimeout is an error to indicate that a receive operation on the
+// channel timed out.
+var ErrRecvTimeout = errors.New("timed out when waiting for value on channel")
+
+const (
+	// DefaultChanRecvTimeout is the default timeout for receive operations on the
+	// underlying channel.
+	DefaultChanRecvTimeout = 1 * time.Second
+	// DefaultChanBufferSize is the default buffer size of the underlying channel.
+	DefaultChanBufferSize = 1
+)
+
+// Channel wraps a generic channel and provides a timed receive operation.
```

Comment:
For extensibility, should this be a context?

Also, it could go into `grpc/internal/testutils`

Reply:
I moved it to `grpc/internal/testutils`. 
But I don't quite understand where you want me to use a context here.